### PR TITLE
Project Survey - Linda Hintze

### PR DESCRIPTION
**Pull request by lindahz**
**Link to my deployed project:**
https://zodiaclovematch.netlify.app/

**My reflections on how this week's project turned out:**
This project went well. I'm a little disappointed that I don't have time to do more than blue level (bootcamp is more time consuming than expected).

**Things I'd like to have clarified or explained in more detail:**
Nothing I can think of this week.

### CODE REVIEW

File: code/src/components/Summary.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <p>See you later, tiger! <span role="img" aria-label="tiger">🐯</span></p>
+          </div>
+        )         
+      }
+      <button
+        className="summary-button"
+        type="button"
+        onClick={refreshPage}>
+        Start over
+      </button>
```
Comment (johannarexin) [inline code/src/components/Summary.js:43]: Good that you can reload the page with a button :)

File: code/src/components/Zodiac.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+
+export const Zodiac = ({setZodiac, zodiac, zodiacArray}) => {
+
+  console.log(zodiac)
```
Comment (johannarexin) [inline code/src/components/Zodiac.js:5]: Delete console.log!

**Review by johannarexin — COMMENTED** [review]
I find your code really clean and organized :) And the site has a really nice design as well! Having a hard time finding something to "change" in the code. Well done 😍

File: code/src/components/Zodiac.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+              </option>
+            )
+          })}
+        </select>
+      </label>
+      <a href="#match">
+        <button 
+          type="button" 
+          className="next-button"
+          disabled={zodiac === ''}>
```
Comment (EmOlsz) [inline code/src/components/Zodiac.js:28]: Or `disabled={!zodiac}`

File: code/src/App.js [hunk 4]
```diff
@@ -1,9 +1,56 @@
-import React from 'react'
+import React, { useState } from 'react'
+import data from './data.json'
+import { Summary } from './components/Summary.js'
```
Comment (EmOlsz) [inline code/src/App.js:3]: There is a convention which says that we should divide outer dependencies (those located in node_modules) from local dependencies (all local files that we created - components, functions, data such as arrays) by one empty line. Moreover, `style.css` should be separated as well and located below local dependencies. Ideally, it should look like this :

```jsx
import React from 'react';
import ReactDOM from 'react-dom';

import MyComponent1 from './components/MyComponent1';
import MyComponent2 from './components/MyComponent2';
import MyComponent3 from './components/MyComponent3';

import './style.css';
```

So, final order is :

1. Outer dependencies
2. Local dependencies
3. Style file(s)

**Comment by EmOlsz** [comment]
Hey Linda! Let's delete all console.logs from your code, after that I will be able to approve your project

**Comment by lindahz** [comment]
Hi, console.log in zodiac component is now deleted.

________________________________
Från: Maksymilian Olszewski <notifications@github.com>
Skickat: den 2 november 2020 12:00
Till: Technigo/project-survey <project-survey@noreply.github.com>
Kopia: lindahz <lindahintze@hotmail.com>; Author <author@noreply.github.com>
Ämne: Re: [Technigo/project-survey] Project Survey - Linda Hintze (#105)


Hey Linda! Let's delete all console.logs from your code, after that I will be able to approve your project

**Pull request closed without merging**